config/imitation/PushBlock.yaml

```diff
@@ -16,16 +16,24 @@ behaviors:
       num_layers: 2
       vis_encode_type: simple
     reward_signals:
-      gail:
+      extrinsic:
         gamma: 0.99
         strength: 1.0
+      gail:
+        gamma: 0.99
```

Comment:
Should this be lower ?

Reply:
I don't believe so. My feeling is that we decrease strength not gamma to curb the effect of GAIL